mimium-audiodriver/src/backends/cpal.rs

```diff
@@ -111,15 +98,15 @@ impl NativeAudioData {
                     for i in 0..dspout {
                         o[i] = res[i] as f32;
                     }
-                    for i in dspout..hout {
-                        o[i] = 0.;
-                    }
                 }
                 (1, 2) => {
                     o[0] = res[0] as f32;
                 }
```

Comment:
This case should probably be covered by the next `(hout, _)` case.

Reply:
Indeed. I was wondering if we should mix stereo signal when the case of mono output hardware && stereo DSP code. For now, I'll simply delete the case above.

Reply:
I see. Either should be fine.